IPJ/IPJ6_PictureRGB256/src/main/java/PictureRGB256.java

```diff
@@ -0,0 +1,41 @@
+import javax.imageio.ImageIO;
+import java.awt.image.BufferedImage;
+import java.io.*;
+import java.io.IOException;
+
+public class PictureRGB256 {
+    final int width = 256;
+    final int height = 256;
```

Comment:
It would make sense, to make this generic; I mean that user might be able to pass those values.
Just Idea, it's your program:)

Reply:
@MatDls i change API - all final values are input data (width, height, error, pathname)